### ISSUE TITLE
[Feature request]:MD.Emu Videoscreen Mode

### ISSUE THREAD

**Issue by TheBoxGame** [Rakashazi/emu-ex-plus-alpha#1674]
### Describe the feature

stretching the screen so that you can see more of what is behind the screen, as in this screenshot

### Add screenshots

![Screenshot_2024-06-08-11-53-35-331_com google android youtube](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/23b54b21-9a4f-4181-92bf-56b0e237a847)

**Comment by TheBoxGame** [comment]
![Screenshot_2024-06-08-15-49-42-234_com retroarch aarch64](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/75b40d5e-e2c4-43ff-99bd-05626a3777a8)

**Comment by TheBoxGame** [comment]
![Screenshot_2024-06-08-16-12-41-623_com retroarch aarch64](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/5bb2de59-b1d0-4a8e-a08f-322fa12becd5)

**Comment by ErickN13** [comment]
Isso seria muito util

**Comment by TheBoxGame** [comment]
@ErickN13 yes

**Comment by TheBoxGame** [comment]
@ErickN13 esta função videscreen permanecerá em md.emu porque sinto falta desta função

**Comment by ErickN13** [comment]
Até agora o único jeito de deixar a tela em 16:9 seria esticando a Tela nas Configurações do Emulador

**Comment by master1274** [comment]
I seriously doubt that this is feasible in the current state. Genesis Plus GX Wide is a fork of the current version Genesis Plus GX.

**Comment by TheBoxGame** [comment]
@master1274 why are there doubts about a videoscreen?

**Comment by master1274** [comment]
Why, this hack is not even in the current version Genesis Plus GX. Only in the fork. A version Genesis Plus GX in the emulator MD.emu generally very outdated. Therefore, you will have to implement this hack (for it to work more or less correctly) completely from scratch. And this is not such an easy task.

**Comment by master1274** [comment]
I often see various requests to improve the emulator MD.emu. Including updates Genesis Plus GX. I will answer immediately to everyone who is waiting for something similar. Expect major changes to this emulator if and only if the developer makes MD.emu free on Google Play. After all, only then the developer MD.emu will be able to update the current version Genesis Plus GX. Which is now licensed - Non-commercial.

**Comment by TheBoxGame** [comment]
@master1274 yes, genesis plus gx is outdated and very much so, but I'm not the one updating it all, justrakshazi and I asked if this option was good for genesis plus gx, i.e. md.emu

**Comment by TheBoxGame** [comment]
and it would also be useful for someone who has been waiting for so long You'll know what I mean, it's marked with pissing marks
![329848544-6f0c7adb-422b-4cf2-ac0c-057a6edf7b36](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/479ec4ef-2411-4aa7-a97d-ee9d913a22d4)

**Comment by master1274** [comment]
The developer can quite possibly implement pruning even at the interface settings level.

**Comment by TheBoxGame** [comment]
@master1274 how

**Comment by TheBoxGame** [comment]
@master1274 hey a have question Do you know C++? If so, could you replace it because not only do I mean these 2 functions, but also the one for nes.emu, i.e. display nsf
![325985842-1e288062-0a3e-4e16-90cf-a3d71f4fb053](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/4c44d92e-bbfa-4780-b816-687637729e98)

**Comment by master1274** [comment]
No. I'm not a programmer and I don't know any programming languages. I'm interested in emulating gaming hardware as a simple user.

**Comment by TheBoxGame** [comment]
> Nie. Nie jestem programistą i nie znam żadnego języka programowania. Jako prosty użytkownik jestem zainteresowany emulacją sprzętu do gier.

ok
